safe/gui/tools/wizard/test/test_keyword_wizard.py

```diff
@@ -320,6 +321,152 @@ def test_hazard_volcano_polygon_keyword(self):
         real_keywords = dialog.get_keywords()
         self.assertDictEqual(real_keywords, expected_keyword)
 
+    def test_hazard_earthquake_polygon_keyword(self):
+        """Test keyword wizard for earthquake hazard polygon."""
```

Comment:
I think you should rename the test to `test_layer_without_inasafe_fields` because this test is not specific to earthquake.